bin/generate-readme

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+transcribe() {
+  local -r project="$(node --print 'require("./package.json").repository.url.replace(/[.]git$/, "").split("/").slice(-2).join("/")')"
+  local -r version="${VERSION?Environment variable not set}"
+  local -r url="https://github.com/$project/blob/v$version/{filename}#L{line}"
```

Comment:
Good point. I had not noticed this unstated assumption. I had been unsure as to whether sanctuary-scripts should manage the xyz dependency; you've made it clear to me that it should.